CHANGELOG.md

```diff
@@ -53,6 +53,9 @@ v0.33.1 (2023-05-01)
 - Fix bug where some capsule values (such as Prometheus receivers) could not
   properly be used as an argument to a module. (@rfratto)
 
+- Return an error message rather than go panic when putting a logging config
+  block inside a module. (@erikbaranowski)
```

Comment:
(btw, this needs to be moved out of the v0.33.1 section) 

```suggestion
- Fix an issue where defining `logging` or `tracing` blocks inside of a module would generate a panic instead of returning an error. (@erikbaranowski)
```